napari/_qt/widgets/_tests/test_shortcut_editor_widget.py

```diff
@@ -118,7 +149,7 @@ def test_keybinding_with_modifiers(
         widget._table.viewport(), Qt.MouseButton.LeftButton, pos=item_pos
     )
     qtbot.waitUntil(lambda: QApplication.focusWidget() is not None)
-    qtbot.keyClicks(QApplication.focusWidget(), key, modifier=modifier)
+    qtbot.keyClick(QApplication.focusWidget(), key, modifier=modifier)
```

Comment:
Well I did this to get access to the line edit widget, so with the mouse movement/interaction above the application focus should end up being the line edit widget used to edit shortcuts for a specific column/row combination. But maybe that should be though in a different way then? Do you know another way to set the focus and get a reference to the lineedit widget used to edit an specific shortcut/table cell?

Reply:
Gave a check locally and indeed I was able to reproduce the test failing when using `xvfb-run` due to a `TimeoutErrot` locally (over a Ubuntu VM). Not totally sure but maybe that could be related with https://github.com/pytest-dev/pytest-qt/issues/206 🤔 and the need to setup a window manager to handle events when running the tests via ``xvfb-run` since over CI the tests are passing?

Reply:
I prefer differnt skip decorator. Many of our test requirews spawn window and I run them locally to not consume CI. Bus spawning windows are distracting and problematic.

Reply:
Makes sense to me :+1: So if I'm understanding correctly, I should create a new skip decorator, something like:

```python
skip_local = pytest.mark.skipif(not os.getenv('CI'), "Test to be run only on CI")
```

And use it on `test_keybinding_with_modifiers`, `test_keybinding_with_only_modifiers`, `test_remove_shortcut` and `test_keybinding_editor_modifier_key_detection` (the tests that use the `QApplication.focusWidget` wait), right?

Reply:
based on `skip_local_popups`

```
not os.getenv('CI') and os.getenv('NAPARI_POPUP_TESTS', '0') == '0',
```

I suggest:

```
not os.getenv('CI') and os.getenv('NAPARI_FOCUS_TESTS', '0') == '0',
```